guides/common/modules/proc_adding-a-default-http-proxy.adoc

```diff
@@ -15,9 +15,10 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-a-default-
 . Optional: If authentication is required, in the *Username* field, enter the username to authenticate with.
 . Optional: If authentication is required, in the *Password* field, enter the password to authenticate with.
 . To test connection to the proxy, click *Test Connection*.
+ifdef::katello,orcharhino,satellite[]
+. Select the *Default content HTTP proxy* option.
+endif::[]
 . Click *Submit*.
-. In the {ProjectWebUI}, navigate to *Administer* > *Settings*, and click the *Content* tab.
-. Set the *Default HTTP Proxy* setting to the created HTTP proxy.
```

Comment:
Can we update the CLI procedure as well?

Reply:
We can't because it isn't implemented in Hammer according to @stejskalleos